src/api-reference/invoice/v3.purchase-order-receipt.markdown

```diff
@@ -0,0 +1,180 @@
+  "PurchaseOrderNumber": "PO12345",
+  "ReceivedDate": "2018-09-15 00:00:00.0",
+  "ReceivedQuantity": "14.5",
+  "UnitOfMeasureCode": "DA",
+  "URI": "string",
+  "Version": "2.0"
+}
+```
+
+### Response  
+
+[Response schema](#schema-response)
+
+## <a name="put-receipts"></a>Update purchase order line item with receipt information  
+
+```
+PUT /api/v3.0/invoice/purchaseorderreceipts
+```
+### Parameters  
+
+Name|Type|Format|Description
+---|---|---|---
+`purchaseOrderReceipt`|-|`body`|Purchase order receipt information.
+
+### Input  
+
+[Receipt schema](#schema-receipt)
+
+### Response  
+
+[Response schema](#schema-response)
+
+
+## <a name="get"></a>Get existing purchase order receipt 
+
+```
+GET /api/v3.0/invoice/purchaseorderreceipts?purchaseOrderNumber=PO1&lineItemExternalID=L1&goodsReceiptNumber=GRN1
+```
+
+### Parameters  
+
+Name|Type|Format|Description
+---|---|---|---
+`purchaseOrderNumber`|`string`|{purchaseOrderNumber}|The purchase order number.
+`lineItemExternalID`|`string`|{lineItemExternalID}|A customer-supplied value that uniquely identifies the line item within the purchase order.
+`goodsReceiptNumber`|`string`|{goodsReceiptNumber}|Goods receipt number for a purchase order line item receipt.
+
+
+### Input  
+
+None.
+
+### Response
+
+[Schema](#get-schema-receipt)
+
+## <a name="delete"></a>Delete existing purchase order receipt
+
+```
+DELETE /api/v3.0/invoice/purchaseorderreceipts?purchaseOrderNumber=PO1&lineItemExternalID=L1&goodsReceiptNumber=GRN1
+```
+
+### Parameters  
+
+Name|Type|Format|Description
+---|---|---|---
+`purchaseOrderNumber`|`string`|{purchaseOrderNumber}|The purchase order number.
+`lineItemExternalID`|`string`|{lineItemExternalID}|A customer-supplied value that uniquely identifies the line item within the purchase order.
+`goodsReceiptNumber`|`string`|{goodsReceiptNumber}|Goods receipt number for a purchase order line item receipt.
+
+
+### Input  
+
+None.
+
+### Response
+
+[Response schema](#schema-response)
+
+## <a name="schema-response"></a>Response schema  
+
+Name|Type|Format|Description
+---|---|---|---
+`ErrorCode`|`string`|-|A code that indicates why the purchase order was not processed successfully.
+`ErrorMessage`|`string`|-|A description of the error.
+`FieldCode`|`string`|-|A code that indicates which field caused an issue. This code typically appears only when a field is being validated against a field of a form type. Format: LEVEL CODE. The possible levels are: Header, ShipTo, BillTo, LineItem, Allocation.
+`LineItemExternalID`|`string`|-|The external ID of a line item that caused an error. If the error is related to an allocation, this field indicates the external ID of the line item that the allocation is associated with, and also indicates the allocation that caused of the error.
+`Message`|`string`|-|
+`PurchaseOrderNumber`|`string`|-|The purchase order number.
+`Status`|`string`|-|The result of processing the purchase order. Format: SUCCESS or FAILURE
```

Comment:
Should this be purchase order receipt instead of purchase order?

Reply:
Fixed in bbd8fe90